clients/drcachesim/tools/func_view.cpp

```diff
@@ -140,6 +140,7 @@ func_view_t::parallel_shard_init_stream(int shard_index, void *worker_data,
     auto shard_data = new shard_data_t;
```

Comment:
> A REGDEPS trace still needs a
> funclist.log in the trace directory containing:
> 4294967498,6,,SYS_futex == ID, #args, unused, name
> in order for func_view to work.

Does `-record_syscall` add an entry to funclist.log?  I didn't think it did.  (Looking back, with issues we've since had internally, that was probably a mistake.). So I don't think a user would have such an entry, right?

Reply:
https://dynamorio.org/sec_drcachesim_funcs.html seems to imply that `-record_syscall` should add an entry to funclist.log, but looking at our internal issues, it looks like it doesn't.
Anyway, I should have specified that I added funclist.log with the SYS_futex entry manually in my testing.
So, to answer your question, a user generating a public trace wouldn't have funclist.log, but my plan was to (manually?) add it to our release of public traces.